lib/kb_trimmomatic/kb_trimmomaticImpl.py

```diff
@@ -848,25 +889,32 @@ def execTrimmomatic(self, ctx, input_params):
                                       #'info':
                                           })
                 if some_unpaired_rev_output_created:
-                    output_readsSet_obj = { 'description': input_readsSet_obj['data']['description']+" Trimmomatic unpaired rev reads",
+                    output_readsSet_obj = { 'description': str(input_readsSet_obj['data']['description'])+" Trimmomatic unpaired rev reads",
                                             'items': items
                                             }
                     output_readsSet_name = str(input_params['output_reads_name'])+'_trimm_unpaired_rev'
                     unpaired_rev_readsSet_ref = setAPI_Client.save_reads_set_v1 ({'workspace_name': input_params['output_ws'],
                                                                                   'output_object_name': output_readsSet_name,
                                                                                   'data': output_readsSet_obj
                                                                                   })['set_ref']
+
+                    unpaired_rev_SampleSet_ref = self._save_RNASeqSampleSet(
+                                                            items,
+                                                            input_params['output_ws'],
+                                                            output_readsSet_name + '_SampleSet',
+                                                            reads_desc_ext,
+                                                            single_reads)
                 else:
                     self.log (console, "no unpaired_rev readsLibraries created")
                     unpaired_rev_readsSet_ref = None
 
 
             # create return output object
-            output = { 'report': report,
-                       'output_filtered_ref': trimmed_readsSet_ref,
-                       'output_unpaired_fwd_ref': unpaired_fwd_readsSet_ref,
-                       'output_unpaired_rev_ref': unpaired_rev_readsSet_ref
-                     }
+            output = {'report': report,
+                      'output_filtered_ref': trimmed_readsSet_ref,
+                      'output_unpaired_fwd_ref': unpaired_fwd_readsSet_ref,
+                      'output_unpaired_rev_ref': unpaired_rev_readsSet_ref
+                    }
```

Comment:
output RNASeqSampleSet object should be included here.

Reply:
👍